admin/translations/fr.json

```
@@ -102,6 +102,12 @@
         "title": "Modification des organisations en masse",
         "description": "Note: Seules les colonnes ayant une valeur seront modifiées.",
         "notifications": {
+          "error": {
```

Dans l'existant des fichiers de traduction on trouve à la fois `"errors": {` et `"error": {`, mais la version au pluriel semble plus logique et c'est aussi celle qui est en plus grand nombre, d'où la suggestion suivante : 

```suggestion
          "errors": {
```